chrome/application-services-navigation.json

```diff
@@ -9,26 +9,51 @@
             "href": "/application-services/overview"
         },
         {
-            "title": "Streams for Apache Kafka",
+            "appId": "applicationServices",
+            "title": "API Management",
+            "href": "/application-services/api-management"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Data Science",
+            "href": "/application-services/data-science"
+        },
+        {
+            "appId": "dbaas",
```

Comment:
For the database access app to work, you also have to update the `fed-mods.json` and `main.yml` files. There are `dbass` entries in the `ci-beta` branch already.

Reply:
@Hyperkid123 I have made the changes you suggested! Please have a look!